modules/dir2000/src/main/java/com/hillel/elementary/javageeks/dir/homework3/merge_sort/MergeSort.java

```diff
@@ -0,0 +1,55 @@
+package com.hillel.elementary.javageeks.dir.homework3.merge_sort;
+
+import java.util.Arrays;
+
+public class MergeSort {
+    static int [] arrayToSort;
```

Comment:
Again. Try avoiding using static fields if it's possible. You won't be able to use this method concurrently